src/components/transformation/create/index.tsx

```diff
@@ -148,26 +147,40 @@ function TransformationCreate() {
                 );
             }
             const draftId: string = draft.data[0].id;
-            const specsByName = await getLiveSpecsByCatalogNames(
-                null,
-                Array.from(selectedCollectionSet)
+            const spec = {
```

Comment:
This function is slowly growing... can we move this over into a stand alone hook we could use?